neurom/core/_neuron.py

```diff
@@ -291,19 +407,64 @@ def __str__(self):
     __repr__ = __str__
 
 
-class Neuron:
+class Neuron(morphio.mut.Morphology):
     """Class representing a simple neuron."""
 
-    def __init__(self, soma=None, neurites=None, sections=None, name='Neuron'):
-        """Initialize a Neuron object."""
-        self.soma = soma
-        self.name = name
-        self.neurites = neurites
-        self.sections = sections
+    def __init__(self, filename, name=None):
+        """Neuron constructor.
+
+        Args:
+            filename (str|Path): a filename
+            name (str): a option neuron name
+        """
+        super().__init__(filename)
+        self.remove_unifurcations()
```

Comment:
mmmh, I know this is needed for most neurom features, but maybe we can put a warning if something was changed? I need to check if a warning is present or if we can add one.

Reply:
Nothing changed here. `remove_unifurcations` happened in NeuroM v1 as well. In v2 it happens via this call.

Reply:
ok, cool, so it should not cause troubles then, thanks for clarifying! So this solves : https://github.com/BlueBrain/NeuroM/issues/872 , right?

Reply:
I don't know to be honest about #872. It is up for you to research. NeuroM v1 merged unifurcations (see https://github.com/BlueBrain/NeuroM/blob/master/neurom/io/datawrapper.py#L209). So, we need to merge them here as well.